### PR TITLE
Added sample code

### PR DESCRIPTION
**Pull request by leastofthese23**
Added sample code, included text on possible benefit of using a method with out.

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/dotnet/docs?pullRequest=28954) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet/docs?pullRequest=28954) before we can accept your contribution.<br/><br/>:x: leastofthese23 [sign now](https://cla.dotnetfoundation.org/dotnet/docs?pullRequest=28954)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet/docs?pullRequest=28954) it.</sub>

File: docs/csharp/language-reference/keywords/out.md [hunk 1]
```diff
@@ -18,6 +18,36 @@ You can use the `out` keyword in two contexts:
 
 - In [generic type parameter declarations](out-generic-modifier.md) for interfaces and delegates, which specifies that a type parameter is covariant.
 
+## Example
+The out keyword can be especially useful when you want to get 2 values back from a method for 2 local variables, without using a custom object, tuple or class-scoped variable. In the below example, the method `IsDivisibleBy2` takes in an integer, determines if it is divisible by 2, returns the result as a `bool` and outs the division result.
```
Comment (BillWagner) [inline docs/csharp/language-reference/keywords/out.md:22]: I'd soften this somewhat:

```suggestion
You can use the `out` keyword to retrieve multiple values back from a method in multiple variables, without using a custom object, tuple or class-scoped variable. In the below example, the method `IsDivisibleBy2` takes in an integer, determines if it is divisible by 2, returns the result as a `bool` and outs the division result.
```

File: docs/csharp/language-reference/keywords/out.md [hunk 2]
```diff
@@ -18,6 +18,36 @@ You can use the `out` keyword in two contexts:
...
+
+   
+    public static bool IsDivisibleBy2(int x, out double dividedBy2)
```
Comment (BillWagner) [inline docs/csharp/language-reference/keywords/out.md:25]: There's a markdown lint warning for the style (indent vs. triple-backtick). Instead, you should add the sample to the associated snippets project, so the new code is validated in our CI system.

**Review by BillWagner — COMMENTED** [review]
hi :wave: @leastofthese23 

There's a few items that we should address before merging this.

First, the text overstates the usefulness of `out` parameters vs. tuples (or other return types). At a minimum, you should remove the "extremely" from "

Second, there are markdown lint warnings.

Third, we put the samples in our snippets project. That way, our sample code runs through the CI build system.

Finally, before we merge this, you'll need to sign the CLA (instructions are in the note from the CLA bot).

File: docs/csharp/language-reference/keywords/out.md (hunk 1 above)
Comment (leastofthese23) [reply docs/csharp/language-reference/keywords/out.md:22]: I agree with you on other points, but disagree with you on this wording. In my opinion the former is better.

File: docs/csharp/language-reference/keywords/out.md (hunk 2 above)
Comment (leastofthese23) [reply docs/csharp/language-reference/keywords/out.md:25]: Where can I find the associated snippets project?

Comment (MSDN-WhiteKnight) [reply docs/csharp/language-reference/keywords/out.md:25]: I think there's none currently, but you could add one somewhere under https://github.com/dotnet/docs/tree/main/samples/snippets/csharp/language-reference/keywords

**Pull request closed without merging**